packages/nimble-components/src/rich-text-viewer/tests/rich-text-viewer.spec.ts

```diff
@@ -10,4 +31,206 @@ describe('RichTextViewer', () => {
+            );
+            expect(lastChildElement!.getAttribute('href')).toBe(
+                'https://nimble.ni.dev/'
+            );
+
+            await disconnect();
+        });
+
+        it('should convert direct links in bold markdown string to "strong" and "a" HTML tags', async () => {
+            element.markdown = '**<https://nimble.ni.dev/>**';
+
+            await connect();
+
+            expect(getChildElements()).toEqual(['P', 'STRONG', 'A']);
+            expect(lastChildElement?.textContent).toBe(
+                'https://nimble.ni.dev/'
+            );
+            expect(lastChildElement!.getAttribute('href')).toBe(
+                'https://nimble.ni.dev/'
+            );
+
+            await disconnect();
+        });
+    });
+
+    describe('various not supported markdown string values render as expected', () => {
+        const notSupportedMarkdownStrings: { name: string }[] = [
+            { name: '> blockquote' },
+            { name: '`code`' },
+            { name: '```fence```' },
+            { name: '# Heading' },
+            { name: '[link](url)' },
+            { name: '[ref][link] [link]:url' },
+            { name: '![Text](Image)' },
+            { name: '&nbsp;' },
+            { name: '---' },
+            { name: '<div><p>text</p></div>' }
+        ];
+
+        const focused: string[] = [];
+        const disabled: string[] = [];
+        for (const value of notSupportedMarkdownStrings) {
+            const specType = getSpecTypeByNamedList(value, focused, disabled);
+            // eslint-disable-next-line @typescript-eslint/no-loop-func
+            specType(
+                `string "${value.name}" renders as plain text "${value.name}" within paragraph tag`,
+                // eslint-disable-next-line @typescript-eslint/no-loop-func
+                async () => {
+                    element.markdown = value.name;
+
+                    await connect();
+
+                    const childElement = pageObject.getFirstChildElement();
+                    expect(childElement?.tagName).toBe('P');
+
+                    await disconnect();
+                }
+            );
+        }
+    });
+
+    describe('various wacky string values render as expected', () => {
+        const focused: string[] = [];
+        const disabled: string[] = [];
+        for (const value of wackyStrings) {
+            const specType = getSpecTypeByNamedList(value, focused, disabled);
+            // eslint-disable-next-line @typescript-eslint/no-loop-func
+            specType(
+                `string "${value.name}" renders as plain text "${value.name}" within paragraph tag`,
+                // eslint-disable-next-line @typescript-eslint/no-loop-func
+                async () => {
+                    element.markdown = value.name;
+
+                    await connect();
+
+                    const childElement = pageObject.getFirstChildElement();
+                    expect(childElement?.tagName).toBe('P');
```

Comment:
Could you also assert that the contents of the `<p>` tag match the expected string `value.name`?

Same for the "not supported markdown string values" test above.